src/pages/home.js

```diff
@@ -18,99 +15,55 @@ const styles = (theme) => ({
   },
 });
 
-const slideData = [
-  {
-    index: 0,
-    headline: "New Fashion",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/fashion.jpg",
-  },
-  {
-    index: 1,
-    headline: "In The Wilderness",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/forest.jpg",
-  },
-  {
-    index: 2,
-    headline: "For Your Current Mood",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/guitar.jpg",
-  },
-  {
-    index: 3,
-    headline: "Focus On The Writing",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/typewriter.jpg",
-  },
-  {
-    index: 4,
-    headline: "New Fashion Trend",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/fashion.jpg",
-  },
-  {
-    index: 5,
-    headline: "In The Wilderness",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/forest.jpg",
-  },
-  {
-    index: 6,
-    headline: "For Your Current Mood",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/guitar.jpg",
-  },
-  {
-    index: 7,
-    headline: "Focus On The Writing",
-    button: "Read Now",
-    src: "https://s3-us-west-2.amazonaws.com/s.cdpn.io/225363/typewriter.jpg",
-  },
-];
-
+const popularBook = [];
+function Book(index, title, author, bookImage) {
+  this.index = index;
+  this.title = title;
+  this.author = author;
+  this.bookImage = bookImage;
+}
 class home extends Component {
+  componentDidMount() {
+    axios
+      .get("https://us-central1-socialape-aa1d6.cloudfunctions.net/api/screams")
```

Comment:
uhh you don't need the whole ""https://us-central1-socialape-aa1d6.cloudfunctions.net/api/screams" here. saying get("screams") is enough.

Also don't call axios here. If you follow the previous format, you should make a function in the actions folder and do the API calls there. (i.e. fetchBooks)

Then in componentDidMount(), call the function that you made in the actions folder  (this.fetchBooks())